definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -4772,5 +4772,40 @@
     "CaseEventID": "IssueAOSOffline_Respondent",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "IssueAOSOffline_CoRespondent",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
```

Comment:
@iichr @jvictorcf 
Shouldn't this be 'CRU' ? Otherwise no one can trigger the event